src/tools/run-moonbeam-fork-solo.ts

```diff
@@ -0,0 +1,221 @@
+// This script is expected to run against a parachain network (using launch.ts script)
```

Comment:
Why not make it a simple argument of the "run-moonbeam-fork" instead of creating a new file ?

Reply:
Sure thing, done. 

I initially thought against extending the original script, thinking you wouldn't the existing script overly complex or being messed with.

Reply:
We could probably split the dev/relay spin up into a separate file to keep the main file small if you think it makes sense